pkg/apmintegration/apm.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package apmintegration
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ava-labs/avalanche-cli/pkg/application"
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+)
+
+// Returns alias
+func AddRepo(app *application.Avalanche, repoURL string, branch string) (string, error) {
+	alias, err := getAlias(repoURL)
+	if err != nil {
+		return "", err
+	}
+
+	if alias == constants.DefaultAvaLabsPackage {
+		ux.Logger.PrintToUser("Avalanche Plugins Core already installed, skipping...")
+		return "", nil
+	}
+
+	fmt.Println("Installing repo")
+
+	return alias, app.Apm.AddRepository(alias, repoURL, branch)
+}
+
+func UpdateRepos(app *application.Avalanche) error {
```

Comment:
Kind of feels like we don't need this function if we're just calling `.Update()` and doing nothing else

Reply:
I added it so that a caller only needs to call this package and doesn't need to call the apm directly.